core/src/components/content/test/basic/index.html

```diff
@@ -30,16 +30,16 @@
           <ion-content id="content-part">
             <f style="height: 50%"></f>
             <p>
-              <ion-button onclick="toggleDisplay('header')" color="secondary">Toggle header</ion-button>
-              <ion-button onclick="toggleDisplay('footer')" color="secondary">Toggle footer</ion-button>
-              <ion-button onclick="toggleDisplay('toolbar2')" color="secondary">Toggle 2nd toolbar</ion-button>
+              <button onclick="toggleDisplay('header')" color="secondary">Toggle header</button>
```

Comment:
Can the `color` properties be removed now that these are `button` elements?

Reply:
Removed in https://github.com/ionic-team/ionic-framework/pull/29422/commits/74d7dd7719335983a82364ba10fb54f9136522ea